src/dapp/utils/api_request_helpers.ts

```diff
@@ -0,0 +1,112 @@
+import { error } from 'console'
+
+//Interfaces
+
+export interface Event_shadow_coordinates {
+    event_id: string
+    lat: number
+    lng: number
+}
+
+// Function that creates an auth header for requests
+export const generateAuthenticationHeader = (): string => {
+    // Retrieve the environment variable
+    const user_email = process.env.USER_EMAIL
+    const user_password = process.env.USER_PASSWORD
+
+    // Merging the environment variables into an Authorization header to send with the request
+    // Exporting that it can be used in various requests
+    // "Authorization": `Basic ${credentials}`
+    const credentials = Buffer.from(`${user_email}:${user_password}`, 'utf-8')
+
+    //Convert the Buffer to a Base64-encoded string
+    const base64Credentials = credentials.toString('base64')
+
+    console.log('Creditals: ' + base64Credentials)
```

Comment:
With this log entry you leak your credentials to the log files.

Reply:
Must fix

Reply:
Had that consol.log as confirmation